src/python/ensembl/io/genomio/genome_stats/compare.py

```diff
@@ -189,17 +190,22 @@ def main() -> None:
     )
     parser.add_argument_src_path("--ncbi_stats", required=True, help="NCBI dataset JSON file")
     parser.add_argument_src_path("--core_stats", required=True, help="Core database JSON file")
+    parser.add_log_arguments(add_log_file=True)
     args = parser.parse_args()
 
+    # Configure and initialise logging
+    init_logging(args.log_level, args.log_file, args.log_file_level)
+    
     try:
         ncbi_stats = get_json(args.ncbi_stats)["reports"][0]
-    except KeyError:
+        logging.info(f"{args.ncbi_stats} JSON .'reports' obtained")
+    except (json.decoder.JSONDecodeError, KeyError):
+        logging.warning(f"{args.ncbi_stats} JSON file is empty")
         ncbi_stats = {}
```

Comment:
```suggestion
    except (json.decoder.JSONDecodeError, KeyError):
        logging.warning(f"{args.ncbi_stats} JSON file is empty")
        ncbi_stats = {}
    else:
        logging.info(f"{args.ncbi_stats} JSON .'reports' obtained")
```
As a note, the info message does not look too explanatory for me, but it may be because I'm not too familiar with what the code does.